daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -404,8 +412,22 @@ createProjectPackageDb lfVersion fps = do
 
 execBuild :: Compiler.Options -> Maybe FilePath -> IO ()
 execBuild options mbOutFile = do
-  execInit
-  execPackageNew options mbOutFile
+    execInit
+    execPackageNew options mbOutFile
+
+-- | Remove any build artifacts if they exist.
+execClean :: IO ()
+execClean = do
+    withProjectRoot $ \_relativize -> do
+        isProject <- doesFileExist projectConfigName
+        if isProject then do
+            removePathForcibly projectPackageDatabase
+            removePathForcibly ".interfaces"
+            removePathForcibly "dist"
```

Comment:
Would prefer if we put everything in one dir...

Reply:
Even better if the dir was `.daml`.

Reply:
Me too. I think that can be done in a separate PR though.